src/pe_reports/home/views.py

```diff
@@ -15,4 +20,6 @@ def index():
 
     Gather data from form and insert into database.
     """
+    LOGGER.info("Made it to home")
```

Comment:
A bit of a nitpick, but is this really an `info`-worthy log message?  It seems to be more of a `debug` type message.
```suggestion
    LOGGER.debug("Made it to home")
```

Feel free to disregard if you disagree.